extensions/wikibase/module/styles/schema-alignment.css

```diff
@@ -268,7 +284,7 @@ input[type="checkbox"].wbs-wikitext-override {
   display: inline-block;
 }
 
-.wbs-entity-contents {
+.wbs-entity-contents, .wbs-entity-input {
```

Comment:
I'd keep this out indeed, as it seems unrelated to the issue.

Reply:
Okay sure I'll change it back to its original position